docs/dev-environment-setup.md

```diff
@@ -243,7 +250,7 @@ cd app && yarn test:e2e --spec cypress/integration/accessibility/*
 pre-commit run --all-files
 ```
 
-If you are impatient and your work is isolated to Javascript, it may be faster to run only the related formatter (`xo`), but it may not catch everything (such as the end-of-file fixer and trailing whitespace):
+If you are impatient and your work is isolated to Javascript, it may be faster to run only the linter and formatter (`eslint` and `prettier`), but it may not catch everything (such as the end-of-file fixer and trailing whitespace):
 
 ```
 yarn format
```

Comment:
To be equivalent to the previous `yarn format`, I think this snippet should be changed to:
```suggestion
yarn lint && yarn format
```